From b2c3d4e5f6a1b2c3d4e5f6a1b2c3d4e5f6a1b2c3 Mon Sep 17 00:00:00 2001
From: OpenBMC Build <openbmc@build>
Date: Wed, 04 Mar 2026 00:00:00 +0000
Subject: [PATCH] arm: dts: aspeed: tiogapass: enable PECI for CPU temperature

TiogaPass uses Intel Xeon D (dual-socket) with PECI for CPU/DIMM
temperature reporting. Enable the ASPEED PECI controller so that
intelcpusensor can read DTS_CPU1/DTS_CPU2 and phosphor-pid-control
can perform proper thermal fan management instead of running failsafe.

---
 arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-tiogapass.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-tiogapass.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-tiogapass.dts
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-tiogapass.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-tiogapass.dts
@@ -59,6 +59,10 @@
 &vuart {
 	/* Host console via LPC COM1 (0x3F8) emulation */
 	status = "okay";
 };

+&peci0 {
+	status = "okay";
+};
+
 &fmc {
